sklearnex/linear_model/coordinate_descent.py

```diff
@@ -15,3 +15,12 @@
 # ===============================================================================
 
 from daal4py.sklearn.linear_model import ElasticNet, Lasso
+from onedal._device_offload import support_usm_ndarray
+
+ElasticNet.fit = support_usm_ndarray(queue_param=False)(ElasticNet.fit)
+ElasticNet.predict = support_usm_ndarray(queue_param=False)(ElasticNet.predict)
+ElasticNet.score = support_usm_ndarray(queue_param=False)(ElasticNet.score)

```

Comment:
While I don't think this decoration way will cause any problem, it creates situation where two classes with one module path and name exist in daal4py and sklearnex.

Reply:
In fact, this situation existed before these changes. With this changes, here only added decoration usm ndarray support

Reply:
@Alexsandruss Doing this way will make errors to our users more difficult to disentangle. The trace of an error will lead through the wrapping which is not related to the module file in which it is defined. If they see any error with the wrapping whatsoever, they will have to really work to find how and where it is included.

Reply:
@icfaust personally I agree with this your proposal. It make sense in case of debug.
Will address and create exact sklearnex estimators with daal4py inheritance.